packages/monitoring/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "url": "https://github.com/googleapis/googleapis",
+  "name": "@google-cloud/monitoring",
+  "version": "0.7.1",
+  "author": "Google Inc.",
+  "description": "Stackdriver Monitoring library for Node.js",
+  "main": "src/index.js",
+  "files": [
+    "src"
+  ],
+  "dependencies": {
+    "google-proto-files": "^0.8.2",
+    "google-gax": "^0.7.0",
+    "extend": "^3.0.0",
+    "lodash.union": "^4.6.0"
```

Comment:
I removed it since it is unnecessary. Filed a [issue](https://github.com/googleapis/packman/issues/144) to get this automated.